helm_helpers_test.go

```diff
@@ -233,7 +233,7 @@ func Test_validateReleaseCharts(t *testing.T) {
 				targetMap[target] = true
 			}
 			if got, msg := validateReleaseCharts(tt.args.apps); got != tt.want {
-				t.Errorf("getReleaseChartName() = %v, want %v , msg: %v", got, tt.want, msg)
+				t.Errorf("validateReleaseCharts() = %v, want %v , msg: %v", got, tt.want, msg)
```

Comment:
Does this change have anything to do with windows ? Is it because the test did not report the correct behavior ?

Reply:
I think the old code is wrong. Would you agree? Especially when debugging a unit test, it says something that is not tested here. Here "validateReleaseCharts" and now it says "validateReleaseCharts" failed if it fails.